### PR TITLE
Add a download button #82 and #158

### PR DESCRIPTION
**Pull request by wp99cp**
This should be merged into dev after #168

addresses #82 and #158

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -251,6 +256,22 @@ def status(uuid):
         status=status_code, mimetype='application/json')
     return response
 
+def __delete_after_delay(base_path: str, uuid: str, delay = 300):
```
Comment (maede97) [inline backend/app.py:259]: Here we sleep for 300s. This is 5m. In the frontend, we display 10m.
I think we should sleep for around 12m (in order to also give the user the opportunity to download at the last second with a somewhat slow internet speed.
So I propose to change to 12*60 = 720.
```suggestion
def __delete_after_delay(base_path: str, uuid: str, delay = 720):
```

File: backend/app.py [hunk 2]
```diff
@@ -251,6 +256,22 @@ def status(uuid):
...
+        logger.info('Waiting for %s seconds before deleting folder %s' % (delay, base_path))
+
+        # Wait for 5 minutes (300 seconds)
```
Comment (maede97) [inline backend/app.py:262]:
```suggestion
        # Wait for 12 minutes (720 seconds)
```

File: backend/app.py (hunk 1 above)
Comment (wp99cp) [reply backend/app.py:259]: sorry saw it to late, will fix it on the dev branch

File: backend/app.py (hunk 2 above)
Comment (wp99cp) [reply backend/app.py:262]: thanks for our feedback!

**Pull request merged**